returnn/training.py

```diff
@@ -11,6 +11,7 @@
 import subprocess as sp
 
 from sisyphus import *
+from sisyphus.delayed_ops import DelayedSlice
```

Comment:
where is this needed? You only change doc strings so far.
```suggestion
```

Reply:
ah yes, did not see that one. Not sure what I used that for...